packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -151,7 +114,6 @@ const LoggedInPlansFeatureActionButton = ( {
 	}
 
 	if ( current && planSlug !== PLAN_P2_FREE ) {
-		// TODO: this block can be another "exception" that stays
 		if ( canPurchaseStorageAddOns && nonDefaultStorageOptionSelected && ! isMonthlyPlan ) {
```

Comment:
I'll take a stab at reworking this to be handled from the `useAction` hook. In a separate PR - was thinking of something like passing `isCurrentPlanUpgradeable` (or something of this nature).. WDYT? 🤔

Reply:
> Well actually, not even that. might move this out completely I think

Yes I believe we can move this out 👍